src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp

```diff
@@ -104,27 +104,36 @@ struct convolution_onednn : typed_primitive_onednn_impl<convolution, dnnl::convo
         return args;
     }
 
+    template <typename T>
+    static void set_activation_zero_points_attr(const std::shared_ptr<dnnl::primitive_attr>& attrs, cldnn::data_node& node) {
+        int32_t zp_val = DNNL_RUNTIME_S32_VAL;
+        bool is_per_tensor = onednn::is_per_tensor<T>(node, zp_val);
+        if (is_per_tensor) {
+            attrs->set_zero_points(DNNL_ARG_SRC, 0, {zp_val});
```

Comment:
`zp_val` has to equal the actual scalar value of the zero point, for `DNNL_RUNTIME_S32_VAL` necessitates a GPU buffer that would provide the value instead of that stub.

Reply:
I've changed onednn::is_per_tensor() function. zp_val is set actual scala value in this function.

Reply:
Ah wait, sorry, I just realized that `is_per_tensor` assigns `zp_val`.

Reply:
Resolved.